config/manifests/ocs-operator/bases/ocs-operator.clusterserviceversion.yaml

```diff
@@ -49,6 +44,11 @@ spec:
       kind: StorageProfile
       name: storageprofiles.ocs.openshift.io
       version: v1
+    - description: StorageRequest is the Schema for the StorageRequests API
+      displayName: Storage Request
+      kind: StorageRequest
+      name: storagerequests.ocs.openshift.io
+      version: v1alpha1
```

Comment:
why this got changed?

Reply:
These are auto-generated changes, I guess it just adjusted based on alphabetical order or so.